editor/src/components/canvas/canvas-strategies/strategies/keyboard-reorder-strategy.spec.browser2.tsx

```diff
@@ -100,6 +107,61 @@ const TestProjectMixedInlineFlow = `
 </div>
 `
 
+const TestProjectWithFragment = `
+    <div
+      style={{
+        backgroundColor: '#aaaaaa33',
+        position: 'absolute',
+        left: 47,
+        top: 89,
+        width: 669,
+        height: 338,
+        display: 'flex',
+        gap: 33,
+      }}
+      data-uid='parent'
+    >
+      <div
+        style={{
+          backgroundColor: '#00abff',
+          width: 123,
+          height: 215,
+          contain: 'layout',
+        }}
+        data-uid='child1'
+      />
+      <React.Fragment>
```

Comment:
you can give this an explicit data-uid! – it'll throw a react warning but so be it.

Reply:
Thanks! I updated the test in https://github.com/concrete-utopia/utopia/pull/3302/commits/6d6101dcc1cf0132fea275c33a86187185426f07